### DISCUSSION TITLE
second input?

### DISCUSSION THREAD

**Discussion by host-felice** [rvdbreemen/OTGW-firmware#249] [Q&A]
Hello,
is it possible to use a second input (as counter) instead of 1 input and 1 output? Maybe compiling the firmware? Or maybe to be able to choose between input and output in a future release?

I am using the nodo shop board with a D1 mini. Are there any pull-up or pull-down resistors?

Thank you!

**Comment by rvdbreemen** [comment]
Hi @Smandurlo 

Currently the firmware can only count one input. I you would like to change that, then you indeed have to create a fork and add the functionality and the UI. 

What is the use-case for a second counter?

Kind regards,
Robert

**Reply by host-felice** [reply]
Hi @rvdbreemen,
Thank you for your quick reply. My use case is as follows: I have two G4 meters. The first one is for the actual boiler with OTGW and the second meter is for another boiler that is two floors above. I will need to use another ESP8266 just for the second meter :-)

With Home Assistant I can manipulate the second input of the first OTGW to be the input of the second OTGW.

In any case, I think it would be useful to be able to configure the free GPIOs as input/output in the UI so it can fit whatever use case people have. Maybe someone has a water meter nearby or keeps the OTGW inside an accessible compartment and wants to monitor when it is opened.

Additionally, it should be clear whether the input/output is using internal or external pull-up/down resistors.
